content/en/Getting started/Pentest objectives/_index.md

```diff
@@ -10,22 +10,19 @@ description: >
 Define what you want our pentesters to test.
 {{% /pageinfo %}}
 
-When you set up a pentest through the UI, you:
+<!-- For content, see https://github.com/cobalthq/cobalt-product-public-docs/blob/main/layouts/shortcodes/big-pic-UI-steps.md -->
```

Comment:
Until the PR is merged, you'll see this file in the following directory: https://github.com/cobalthq/cobalt-product-public-docs/tree/DOCS-70_incorporate_5steps_article_from_zendesk/layouts/shortcodes